### ISSUE TITLE
Compare behavior of filter-abund -V with normalize-by-median; maybe adjust defaults

### ISSUE THREAD

**Issue by ctb** [ged-lab/khmer#989]
@drtamermansour makes the good point that the two-step process

normalize-by-median -C 20 ...
filter-abund -V ...

may fail to trim many sequences because normalize-by-median rejects sequences with median k-mer abundance >= 20, and filter-abund -V only trims sequences with median k-mer abundance >= 20.

Separately, it might also be a good idea to make the behavior of filter-abund more conservative by setting -V as the default; this would improve user experience (ref #732) by having the default behavior work ok for all data sets, with "worse than optimal" behavior only on genomic data.

**Comment by ctb** [comment]
This is an issue @standage should read.

**Comment by standage** [comment]
Yes, this is something I've been thinking about. Glad I'm not the only one!
